docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,71 @@
+.. _arch_overview_load_balancing_slow_start:
+
+Slow start mode
+===============
+
+Slow start mode is a configuration setting in Envoy to progressively increase amount of traffic for newly added upstream endpoints.
+With no slow start enabled Envoy would send a proportional amount of traffic to new upstream ednpoints.
+This could be undesirable for services that require warm up time to serve full production load and could result in request timeouts, loss of data and deteriorated user experience.
+
+Slow start mode is a mechanism that affects load balancing weight of upstream endpoints and can be configured per upstream cluster. 
+Currently, slow start is supported in Round Robin and Least Request load balancer types.
```

Comment:
Do we fail the config if configured on other types currently?

Reply:
have not wrapped up this part of code yet, but RFC suggests that loading of config should fail if slow start is configured for unsupported lb type.

Reply:
this is no longer a case, as slow start config is included into specific lb type config (instead of being part of common lb config)